packages/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.0+18
```

Comment:
Should be 0.6.1, same in pubspec
https://dart.dev/tools/pub/versioning#semantic-versions

Reply:
I've updated pubspec and changelog with a new version.